src/HearThis/Utils/NonFatalProblem.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
```

Comment:
Could be. I wrote most/all of the error handling stuff in palaso, and feel that it just keeps accreting, layer upon layer. Maybe that's ok? But I'm uncomfortable continuing it. Given my druthers, I'd make a little library just for error handling.  In any case, this bug has been reported an un-dealt with for years now. I don't think it makes sense to grow the fixing of it into a big architectural task.